src/features/content-answers/ContentAnswersModal.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
 };
 
 const ContentAnswersModal = ({ file, isOpen, onRequestClose }: Props) => {
+    const fileName = file && file.name;
```

Comment:
[nit]
```suggestion
    const fileName = file?.name;
```

Or is it guaranteed? because it looks like a required prop that is passed down to ContentAnswersModalContent

Reply:
guaranteed if we have the file